development-vm/Procfile

```diff
@@ -42,8 +42,6 @@ asset-manager:         govuk_setenv asset-manager         ./run_in.sh ../../asse
 asset-manager-worker:  govuk_setenv asset-manager         ./run_in.sh ../../asset-manager  bundle exec rake jobs:work
 # limelight used port 3040
 # transaction_wrappers used port 3041
-govuk-delivery:        govuk_setenv govuk-delivery        ./run_in.sh ../../govuk-delivery ./startup.sh # govuk-delivery uses port 3042
-govuk-delivery-worker: govuk_setenv govuk-delivery        ./run_in.sh ../../govuk-delivery ./venv/bin/celery worker -A service
```

Comment:
Rather than removing entries completely from this file, previously we've just said `# _app_name_ used port _n_`. I don't remember why this is, but it still seems to be convention (`need_api` was recently removed and has done this).

Reply:
Thanks @issyl0 . Fixed in fe1d8f0be